tests/test_misc.py

```diff
@@ -1867,6 +1867,9 @@ def test_signmessage(node_factory):
     checknokey = l2.rpc.checkmessage(message="message for you", zbase=zm)
     assert checknokey['pubkey'] == l1.info['id']
     assert checknokey['verified']
+    # check that checkmassage used with a wrong zbase format throws an RPC exception
+    with pytest.raises(RpcError):
+        l2.rpc.checkmessage(message="wrong zbase format", zbase="wrong zbase format")
```

Comment:
Right! I avoid this usually because changing the error message required to change the test, but I did not propose a better solution so I introduce the match on the exception message!

Thanks!